q.js

```diff
@@ -34,7 +34,7 @@
 			});
 	};
 
-	var regexUnsafeSymbols = /[\0-\x1F"-\),\.:-@\[-\^`\{-\uFFFF]/g;
+	var regexUnsafeSymbols = /[\0-\x1F"-\),\.:-@\[-_`\{-\uFFFF]/g;
```

Comment:
Did you mean to remove `\^`?

Reply:
Let me re-review.

Reply:
Finally getting back to this. Yes, that was deliberate!

It isn't actually removing the caret `^` because you had the character range `[` through `^` (U+005B to U+005E) and by substituting the `_` (U+005F) the range is now U+005B to U+005F, still including U+005E. 

Of course the character range

```
\[-_
```

could be further simplified to 

```
\[-`
````

(U+005B to U+0060, inclusive) but was trying to be unintrusive.

Reply:
Checking in on this.